src/js/health-records/containers/HealthRecordsApp.jsx

```diff
@@ -6,44 +6,32 @@ import DowntimeNotification, { services } from '../../common/containers/Downtime
 import Modal from '../../common/components/Modal';
 import MHVApp from '../../common/containers/MHVApp';
 import RequiredLoginView from '../../common/components/RequiredLoginView';
-import { mhvAccessError } from '../../common/utils/error-messages';
 import { closeModal } from '../actions/modal';
 import Breadcrumbs from '../components/Breadcrumbs';
 
-// This needs to be a React component for RequiredLoginView to pass down
-// the isDataAvailable prop, which is only passed on failure.
-function AppContent({ children, isDataAvailable }) {
-  const unregistered = isDataAvailable === false;
-  let view;
+const SERVICE_REQUIRED = 'health-records';
 
-  if (unregistered) {
-    view = mhvAccessError;
-  } else {
-    view = children;
-  }
-
-  return (
-    <div className="bb-app">
-      <div className="row">
-        <div className="columns small-12">
-          {view}
-        </div>
+const AppContent = ({ children }) => (
+  <div className="bb-app">
+    <div className="row">
+      <div className="columns small-12">
+        {children}
       </div>
     </div>
-  );
-}
+  </div>
+);
 
 export class HealthRecordsApp extends React.Component {
   render() {
     return (
       <RequiredLoginView
         verify
-        serviceRequired="health-records"
+        serviceRequired={SERVICE_REQUIRED}
         user={this.props.user}>
         <DowntimeNotification appTitle="health records tool" dependencies={[services.mhv]}>
           <AppContent>
             <Breadcrumbs location={this.props.location}/>
-            <MHVApp>
+            <MHVApp serviceRequired={SERVICE_REQUIRED}>
```

Comment:
If I end up implementing passing error handler components to `RequiredLoginView`, we could pass `MHVApp` as that prop and inherit the `serviceRequired` prop from it. At which point we would only care about any of the account creation rigamarole if and only if the required service was not available in the user's list of available services in the first place.